test/test_infixowl.py

```diff
@@ -0,0 +1,375 @@
+from rdflib import logger, BNode, Graph, Literal, Namespace, RDFS, OWL, XSD
+from rdflib.namespace import NamespaceManager
+from rdflib.util import first
+from rdflib.extras.infixowl import (
+    BooleanClass,
+    Class,
+    Collection,
+    DeepClassClear,
+    Individual,
+    some,
+    EnumeratedClass,
+    Property,
+    Restriction,
+    max,
+)
+
+
+# @pytest.mark.skip
+def test_infix_owl_example1():
+
+    exNs = Namespace("http://example.com/")
+
+    namespace_manager = NamespaceManager(Graph())
+    namespace_manager.bind("ex", exNs, override=False)
+    namespace_manager.bind("owl", OWL, override=False)
+    g = Graph()
+    g.namespace_manager = namespace_manager
+
+    Individual.factoryGraph = g
+    classD = Class(exNs.D)
+    anonClass = exNs.someProp << some >> classD
+
+    assert repr(anonClass) == "( ex:someProp SOME ex:D )"
+
+    a = Class(exNs.Opera, graph=g)
+
+    # Now we can assert rdfs:subClassOf and owl:equivalentClass relationships
+    # (in the underlying graph) with other classes using the 'subClassOf'
+    # and 'equivalentClass' descriptors which can be set to a list
+    # of objects for the corresponding predicates.
+
+    a.subClassOf = [exNs.MusicalWork]
+
+    # We can then access the rdfs:subClassOf relationships
+
+    assert repr(list(a.subClassOf)) == "[Class: ex:MusicalWork ]"
```

Comment:
I think technically `__repr__` is doing the wrong thing here as far as this is concerned: https://docs.python.org/3/reference/datamodel.html#object.__repr__

Fine to test for what it is doing but probably we should fix it in a major release.

Reply:
If you agree please let me know, I will make an issue.

Reply:
I got derailed, `str` is what I _should_ have been using. Thanks for your unfailingly reliable backstop, it's much appreciated.